src/main/java/hcfsfuse/fuse/HCFSFuseFileSystem.java

```diff
@@ -104,6 +105,7 @@ public HCFSFuseFileSystem(FileSystem fs, FuseOptions fuseOptions,
         .build(new PathCacheLoader());
     mOpenFiles = new IndexedSet<>(ID_INDEX, PATH_INDEX);
     mConfiguration = conf;
+    openoncurrent = mConfiguration.getInt("hcfs.fuse.jnr.open.file.concurrent", 100);
```

Comment:
Use JNR_OPEN_FILE_CONCURRENT instead of use the string directly.

Reply:
Define a constant Integer.Max in `Constants.java`, to keep consistency to the original behavior.